qp_shogun/shogun/shogun.py

```diff
@@ -123,18 +123,19 @@ def generate_shogun_redist_commands(profile_dir, temp_dir,
     return cmds, output
 
 
-def generate_biom_conversion_commands(input_fp, output_dir, level, version):
-    cmds = []
-    output = join(output_dir, 'otu_table.%s.%s.biom' % (level, version))
-    cmds.append(
-        'biom convert -i {input} '
-        '-o {output} '
-        '--table-type="OTU table" '
-        '--to-hdf5'.format(
-            input=input_fp,
-            output=output))
+def run_shogun_to_biom(in_fp, biom_in, out_dir, level, version):
+    if version == 'redist':
+        output_fp = join(out_dir, 'otu_table.%s.%s.biom'
+                         % (level, version))
+    else:
```

Comment:
There are only 2 options for version, right? Should this test if other and error? Something like:
```
if version == 'redist':
    ...
elif version == 'func':
    ...
else:
   raise ... 
```

Reply:
Makes sense to me.